ONSdigital/eq-author
# Digeq eq 20 survey author foundation

__What__
Author foundation. Django site, connected to Postgres with welcome page and Healthcheck page, includes basic unit tests

__How to test__
1. Install postgres
2. Setup qBuilder database and user eq-author
3. Checkout branch
4. Create a Virtual Environment
5. Install requirements: pip install -r requirements.txt
6. Run database migration: python manage.py migrate
7. Run tests  python manage.py test
8. Run application  python manage.py runserver
9. Check the welcome page and healthcheck pages exist (/app/ and /app/healthcheck)

__Who can test__
Anyone but @LJBabbage
